configuration/concepts/breast_examination.csv

```diff
@@ -0,0 +1,23 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:800
+0c510fac-1318-4e27-8eab-27cbb3a548ae,,,Results,Results,Resultados,Resultados,,Misc,Text,,,,,,,,,,,
+0fa92108-224f-472c-b4de-3a9bf1743d21,,,Dates,Dates,fechas,fechas,,Misc,Text,,,,,,,,,,,
+e3a3ff33-583f-4a4e-b2ab-3e95ccbb24f2,,,Cyclic,Cyclic,Cíclico,Cíclico,,Misc,N/A,,,,,,,,,,,
+e21bc895-e3c5-429f-85cb-2c9524edd0fa,,,Non Cyclic,Non Cyclic,No cíclico,No cíclico,,Misc,N/A,,,,,,,,,,,
+c23d4502-1798-42ad-9cb6-e35d94e782a9,,,Breast Lump,Breast Lump,Bulto en el pecho,Bulto en el pecho,,Misc,Coded,,Yes; No,,,,,,,,,
```

Comment:
Let's make the FSN as 'Presence of Breast Lump' and keep the SSN as it is

Reply:
Let's add a concept called 'Anatomical position' FSN and SSN and 'Body side' coded with answer 'Right/Left'. Add this to the csv called commons.csv

Reply:
Done